### PR TITLE
SAMBRO: tweaks

### PR DESCRIPTION
**Pull request by biplovbhandari**
@flavour Please merge

### CODE REVIEW

File: static/formats/cap/export.xsl [hunk 1]
```diff
@@ -411,8 +411,7 @@
                     <xsl:value-of select="data[@field='lon']" />
                 </xsl:variable>
                 <circle>
-                    <!-- Convert radius from m to km -->
```
Comment (flavour) [inline static/formats/cap/export.xsl]: Why was this removed?
has the CAP std changed?
Internally Sahana GIS uses m, my understanding was that CAP used km, hence the need to multiply by 1000

File: static/formats/cap/import.xsl [hunk 2]
```diff
@@ -457,8 +462,7 @@
                     </data>
                     
                     <data field="radius">
-                        <!-- Radius comes in as km, so convert to m -->
```
Comment (flavour) [inline static/formats/cap/import.xsl]: As-above

File: static/formats/rss/modules/cap.xsl [hunk 3]
```diff
@@ -8,7 +8,7 @@
     <!-- cap_info -->
     <!-- @ToDo: Handle multiple info -->
     <xsl:template match="resource[@name='cap_alert']" mode="contents">
-        <xsl:if test="./data[@field='is_template' and @value='false']">
+        <xsl:if test="./data[@field='is_template' and @value='false'] and ./data[@field='external' and @value='false']">
```
Comment (flavour) [inline static/formats/rss/modules/cap.xsl]: Should this be OR not AND?

File: static/formats/cap/import.xsl [hunk 4]
```diff
@@ -436,7 +441,7 @@
         <resource name="cap_area_location">
             <reference field="location_id" resource="gis_location">
                 <resource name="gis_location">
-                    <data field="gis_feature_type" value="1">Point</data>
+                    <data field="gis_feature_type" value="3">Polygon</data>
```
Comment (flavour) [inline static/formats/cap/import.xsl]: Why change to a Polygon here?
As I see it we are importing Lat/Lon (& maybe Radius) hence these are Points.
Currently we are not importing Polygons
Sure we should probably enhance to import Polygons where-defined, but just changing this to Polygon doesn't achieve that...

**Comment by biplovbhandari** [comment]
@flavour Done...Please merge
